userdocs/userdocs.docc/Articles/Features/automatic-task-creation.md

```diff
@@ -0,0 +1,13 @@
+# Automatic Task Creation
+
+Tasks added for common operations with your Package.
```

Comment:
Ideally this is an imperative sentence highlighting why you'd want to read through "Automatic Task Creation" - often a key task, so maybe:

```suggestion
Add tasks for common operations with your Package.
```